frontend/src/pages/Navbar/RightNavbarNavigation.js

```diff
@@ -32,13 +32,15 @@ const RightNavbarNavigations = ({
   const searchDisabled = history.location.pathname !== "/projects";
```

Comment:
rename it to searchVisible
```suggestion
  const searchVisible= history.location.pathname === "/projects";
```